analysis_idacs.tex

```diff
@@ -0,0 +1,8 @@
+\textbf{I-201} Retrieve PSF cutouts from coadd images (e.g., using the Notebook Aspect of the RSP) to derive parameters of PSF models.
```

Comment:
Rewrite to focus on GPU, the IDAC aspect. This IDAC use-case starts with using the RSP, which is confusing.

Reply:
Confirmed resolved.